packages/monaco-editor/README.md

```diff
@@ -0,0 +1,81 @@
+# @uifabric/monaco-editor
+
+Version of [`monaco-editor`](https://www.npmjs.com/package/monaco-editor) with the CSS files transformed to JS which can be loaded onto the page using `@microsoft/load-themed-styles`, plus some Webpack configuration helpers.
+
+## Package contents
+
+- `esm/*`: Copy of `monaco-editor/esm/*` but with the CSS files transformed. (This package doesn't include the AMD variant since it has a built-in CSS loader.)
+- `monaco-typescript.d.ts`: Proper typings for various things used/returned by `monaco.typescript`, including `TypeScriptWorker` (generated by cloning `monaco-typescript`, building, and merging `release/esm/*.d.ts`)
+- `scripts/addMonacoWebpackConfig.js`: Add Monaco-related webpack configuration
+- `lib/configureEnvironment.js`: Set up `MonacoEnvironment` runtime config
+
+## How to use with Webpack
+
+### 1. Webpack configuration
+
+In your `webpack.config.js`, call the `addMonacoWebpackConfig` helper to add Monaco-related configuration (note that it returns a new configuration object):
+
+```js
+const { addMonacoWebpackConfig } = require('@uifabric/monaco-editor/scripts/addMonacoWebpackConfig');
+
+// Somewhere in this file, call:
+config = addMonacoWebpackConfig(config, includeAllLanguages);
```

Comment:
I think you should put a better illustration of a minimal config here:

```js
const { addMonacoWebpackConfig } = require('@uifabric/monaco-editor/scripts/addMonacoWebpackConfig');
const includeAllLanguage = true;
module.exports = addMonacoWebpackConfig({ ... }, includeAllLanguage);
```

Reply:
What do you think I should include? I'm not sure that having an actual config object in the example would be very helpful because it kind of obscures the point of the code snippet (the point is to pass through your existing webpack config, whatever it is) and creates the risk that people will think they *must* use a config that looks like the one in the example.